### PR TITLE
Convert MISSING from RST to MD file

### PR DESCRIPTION
**Pull request by paulbalandan**

### CODE REVIEW

**Comment by kenjis** [comment]
This PR is difficult to review. It seems this is not only a conversion, but also corrections.

**Comment by paulbalandan** [comment]
I'm not sure what you mean but converting RST to MD is not straight forward. But here it is, commits broken down.

File: MISSING.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
+| az     | Azerbaijani | Qarakom           | Sübhan Cabbarov
+| bn     | Bengali     | kabircse          | Kabir Hossain
+| bg     | Bulgarian   | ivantcholakov     | Ivan Tcholakov
+| ca     | Catalan     | Ignasimg          | Ignasi Molsosa
+| hr     | Croatian    | PaleRider981      | Mario Ljubicic
+| da     | Danish      | janhansson*       | Jan Hansson
+| fil    | Filipino    | dzerium           | Edzer Josh V. Padilla ... See note below
+| gu     | Gujarati    | sajid19991        | Sajid Mathupotra
+| hi     | Hindi       | shindesushilkumar | Sushilkumar Shinde
+| hu     | Hungarian   | orionstar         | László Bóra
```
Comment (kenjis) [inline MISSING.md]: `hu` is added here.

File: MISSING.md [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
-| az     | Azerbaijani | Qarakom           | Sübhan Cabbarov
-| bn     | Bengali     | kabircse          | Kabir Hossain
-| bg     | Bulgarian   | ivantcholakov     | Ivan Tcholakov
-| ca     | Catalan     | Ignasimg          | Ignasi Molsosa
-| hr     | Croatian    | PaleRider981      | Mario Ljubicic
-| da     | Danish      | janhansson*       | Jan Hansson
-| fil    | Filipino    | dzerium           | Edzer Josh V. Padilla ... See note below
-| gu     | Gujarati    | sajid19991        | Sajid Mathupotra
-| hi     | Hindi       | shindesushilkumar | Sushilkumar Shinde
-| hu     | Hungarian   | orionstar         | László Bóra
```
Comment (kenjis) [inline MISSING.md]: And `hu` is removed.

**Review by kenjis — APPROVED** [review]

**Pull request merged**